packages/vsce/src/webviews/resource-inspector-panel/App.tsx

```diff
@@ -0,0 +1,121 @@
+      const data = event.data.data;
+      console.log("printing event: ", data);
+      setLabel(data.label);
+      setAttr(data.attributes);
+      setResource(data.resource);
+      setDetails(data.details);
+    };
+    vscode.addVscMessageListener(listener);
+
+    vscode.postVscMessage({ command: "init" });
+
+    return () => {
+      vscode.removeVscMessageListener(listener);
+    };
+  }, []);
+
+  const attributesMap: Map<string, string> = new Map(Object.entries(attr));
+  const detailsMap: Map<string, string> = new Map(Object.entries(details));
+
+  return (
+    <div className="maindiv">
+      <VscodeTable>
+        <VscodeTableHeader>
+          <VscodeTableRow>
+            <VscodeTableHeaderCell className="header-cell-1">
+              <div className="div-display-1">{(label + "").replace(/Closed|Disabled|Unenabled|\(|\)/g, "")}</div>
+              <div className="div-display-1 div-display-2">{details.status + ""}</div>
+            </VscodeTableHeaderCell>
+          </VscodeTableRow>
+        </VscodeTableHeader>
+        <VscodeTableBody>
+          <VscodeTableCell className="padding-left-20">
+            {Array.from(detailsMap)
+              .filter(([key]) => {
+                return key !== "status";
+              })
+              .map(([key, value]) => (
+                <p className="line">
+                  {key} : {value}
+                </p>
+              ))}
+          </VscodeTableCell>
+        </VscodeTableBody>
+      </VscodeTable>
+
+      <VscodeTable zebra={true} columns={["30%", "70%"]}>
+        <VscodeTableHeader>
+          <VscodeTableRow>
+            <VscodeTableHeaderCell className="padding-right-10">Attributes</VscodeTableHeaderCell>
+            <VscodeTableHeaderCell className="padding-right-10">
+              <div>
+                <div className="div-display-1">Values</div>
+                <VscodeTextfield
+                  type="text"
+                  placeholder="Keyword search..."
+                  onInput={(e: { target: HTMLInputElement }) => setSearch(e.target.value)}
+                  value={search}
+                  className="search-style div-display-1"
+                ></VscodeTextfield>
+              </div>
+            </VscodeTableHeaderCell>
+          </VscodeTableRow>
+        </VscodeTableHeader>
+        <VscodeTableBody>
+          {Array.from(attributesMap)
+            .filter(([key, value]) => {
+              return (
+                (search.toLowerCase() === "" ? key : key.toLowerCase().includes(search) || value.toLowerCase().includes(search)) && key !== "_keydata"
+              );
+            })
+            .map(([key, value]) => (
+              <VscodeTableRow>
+                <VscodeTableCell className="padding-left-20">{key}</VscodeTableCell>
+                <VscodeTableCell className="padding-right-75">{value}</VscodeTableCell>
+              </VscodeTableRow>
+            ))}
```

Comment:
Same down here with the object.
```suggestion
          {Object.entries(riArgs.attributes)
            .filter(([key, value]) => key.toLowerCase() !== "_keydata")
            .filter(
              ([key, value]) =>
                key.toLowerCase().trim().includes(search.toLowerCase().trim()) ||
                value.toLowerCase().trim().includes(search.toLowerCase().trim())
            )
            .map(([key, value]) => (
              <VscodeTableRow>
                <VscodeTableCell className="padding-left-20">{key}</VscodeTableCell>
                <VscodeTableCell className="padding-right-75">{value}</VscodeTableCell>
              </VscodeTableRow>
            ))}
```